inductiva/molecules/examples.py

```diff
@@ -1,23 +1,22 @@
 """Functions to download and load example molecules for GROMACS simulations 
 from the RCSB PDB database."""
-import requests
-from inductiva.utils import optional_deps
+import urllib
+
 
-@optional_deps.needs_molecules_extra_deps
 def load_hemoglobin():
```

Comment:
if we're downloading a file onto the user's computer, we should allow an optional argument for the user to decide where to store the file. We should probably also be using the `inductiva.working_dir` global var